docs/docs/clients/client-side/android.md

```diff
@@ -5,35 +5,18 @@ description: Manage your Feature Flags and Remote Config in your Android applica
 slug: /clients/android
 ---
 
+import CodeBlock from '@theme/CodeBlock'; import { AndroidVersion } from '@site/src/components/SdkVersions.js';
+
 This SDK can be used for Android applications written in Kotlin. The source code for the client is available on
 [GitHub](https://github.com/Flagsmith/flagsmith-kotlin-android-client/).
 
 ## Installation
 
 ### Gradle - App
 
-In your project path `app/build.gradle` add a new dependence
-
-```groovy
-//flagsmith
-implementation("com.github.Flagsmith:flagsmith-kotlin-android-client:v1.6.1")
-```
-
-You should be able to find the latest version in the
-[releases section](https://github.com/Flagsmith/flagsmith-kotlin-android-client/releases) of the GitHub repository.
-
-### Gradle - Project
+In your project path `app/build.gradle` add a new dependency:
 
-In the new Gradle version 7+ update your `settings.gradle` file to include JitPack if you haven't already
-
-```groovy
-repositories {
-    google()
-    mavenCentral()
-
-    maven("https://jitpack.io")
-}
-```
```

Comment:
We can't remove this yet, unless we change the below to `io.flagsmith` - I'm still working on finalising the deprecation of jitpack.

Reply:
Changed to use `io.flagsmith` - jitpack is more friction and can be a blocker as we've seen in the past

Reply:
Hmm... but I'm hoping that `io.flagsmith` is temporary and we will (in the not too distant future, when I find time to understand the migration process for our java client) move to `com.flagsmith`. 

I think the move from jitpack is a bigger change than just hiding it in with these changes.

Reply:
The [`com.flagsmith` release](https://mvnrepository.com/artifact/io.flagsmith.flagsmith-kotlin-android-client/flagsmith-kotlin-android-client) is still on 1.6.1 which does not include `getIdentityFlags`. I'd prefer to ship with `io.flagsmith` even if it's temporary since it has all the latest features and customers are not blocked if they need a bugfix before we can sort out publishing to `com.flagsmith`. We can figure out how to publish to `com.flagsmith` later and it's a trivial change for consumers if they eventually want to migrate.

Reply:
> The [`com.flagsmith` release](https://mvnrepository.com/artifact/io.flagsmith.flagsmith-kotlin-android-client/flagsmith-kotlin-android-client) is still on 1.6.1 which does not include getIdentityFlags

Sorry, I'm confused. The link you included isn't to the `com.flagsmith` (because there isn't one in maven central yet!), it's to the one where I had incorrectly included the name of the package in the namespace (see [here](https://github.com/Flagsmith/flagsmith-kotlin-android-client/pull/57/commits/ef6c30034063dc808649927481d987aa757ae4f7#diff-458e7f80b7f7dc9707ace9d8e9e81c049f8ea426e1bffa418e5b084c29a9dffeR199)). 

To be clear, there is only the io.flagsmith releases in maven central [here](https://central.sonatype.com/artifact/io.flagsmith/flagsmith-kotlin-android-client/versions) and the jitpack releases are just pulled straight from Github releases.

I'm not sure it's worth continuing to argue this though, we can go with what you have here if you think it's the right thing to do.